src/app/wallets/send-on-chain.ts

```diff
@@ -359,23 +372,47 @@ const executePaymentViaOnChain = async ({
           return txHash
         }
 
-        let totalFee: Satoshis
+        let minerFee: Satoshis
 
-        const minerFee = await onChainService.lookupOnChainFee({
+        const minerFee_ = await onChainService.lookupOnChainFee({
```

Comment:
I don't understand this. Higher up you are calling `const estimatedFee = await getFeeEstimate()` which is already calling
```
  const minerFee = await onChainService.getOnChainFeeEstimate({
    amount: amountChecked,
    address,
    targetConfirmations: targetConfsChecked,
  })
```
in its code path - why the duplicate lookups?

Reply:
the first is an estimation, the second is the real fees. the issue is because we don't know in advance how much fees we're really going to pay. this is a limitation in how we use lnd currently. we only have an estimation on the fees that are doing to be paid, but can't have certainty on it.

the way to solve this would be to use psbt but this would implies a lot of the logic would have to be a on our side. this is not advisable to do it now, IMO should be done when we do a refactoring of our send onchain payment stack with a queue to do batch payment